package.json

```diff
@@ -5,11 +5,13 @@
   "main": "./build/src/index.js",
   "bin": "./build/src/bin/release-please.js",
   "scripts": {
-    "test": "c8 --reporter=html --reporter=text mocha --timeout=5000 build/test",
-    "test:integration": "mocha --timeout=5000 build/test-integration",
-    "test:all": "c8 --reporter=html --reporter=text mocha --teimout=5000 build/test-integration build/test",
+    "test": "cross-env ENVIRONMENT=test c8 --reporter=html --reporter=text mocha --recursive --timeout=5000 build/test",
+    "test:integration": "cross-env ENV=test mocha --timeout=5000 build/test-integration",
```

Comment:
So usually, we use `system-test` for the end to end.  Just for the sake of consistency, it would be nice to use the same here.